backend/tests/integration-async/records/indexer/test_literature.py

```diff
@@ -31,12 +30,20 @@ def assert_record():
     retry_until_pass(assert_record, retry_interval=3)
 
 
-def assert_es_hits_count(expected_hits_count):
+def assert_record_in_es(recid):
     def assert_hits():
         current_search.flush_and_refresh("records-hep")
-        result = es_search("records-hep")
-        result_total = get_value(result, "hits.total.value")
-        assert expected_hits_count == result_total
+        hits = LiteratureSearch().query_from_iq(f"recid:{recid}").execute().hits
+        assert hits
+
+    retry_until_pass(assert_hits, retry_interval=5)
+
+
+def assert_record_not_in_es(recid):
+    def assert_hits():
+        current_search.flush_and_refresh("records-hep")
+        hits = LiteratureSearch().query_from_iq(f"recid:{recid}").execute().hits
+        assert not hits
```

Comment:
this will always be `false` no? What is the structure of `hits`?

Reply:
It's a list of `hits` so if es doesn't find anything matching query it's empty

Reply:
And I think it should always be empty considering that recid is unique